### PR TITLE
Feature/shashank/26 09 2021/node changes

### PR DESCRIPTION
**Pull request by shashankmotruri**
Packages Added : node-fetch , path , fs , googleapis , mongodb , Blob

### CODE REVIEW

File: Title 1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+"Logline \n\nTheme : Theme 1\nSub-Genre : Sub-Genre 1\nTitle : Title 1\n"
```
Comment (meharajM) [inline Title 1.txt:1]: unwanted file

File: app/containers/App/index.js [hunk 2]
```diff
@@ -10,18 +10,14 @@
 import React from 'react';
 import { Switch, Route, Router } from 'react-router-dom';
 
-<<<<<<< HEAD
-import HomePage from 'containers/HomePage/index.js';
-=======
 import Dashboard from 'containers/Dashboard/Loadable';
->>>>>>> d8576f96369adda8de79190a7ccfd0934fefe89c
```
Comment (meharajM) [inline app/containers/App/index.js:17]: resolve conflicts

File: app/containers/Dashboard/index.js [hunk 3]
```diff
@@ -25,73 +25,97 @@ import history from '../../utils/history';
 import { Link } from 'react-router-dom';
 
 export default function Dashboard() {
+  const [stories, setStories] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:5000/api/dashboard/dashboardInfo')
```
Comment (meharajM) [inline app/containers/Dashboard/index.js:31]: lets keep all api calls in a separate file.

File: app/containers/Dashboard/index.js [hunk 4]
```diff
@@ -25,73 +25,97 @@ import history from '../../utils/history';
...
-              <h3>Description</h3>
-              <br /><br />
-              <Button
-                type="primary"
-                danger
-              >
-                Logout  
-              </Button>
-            </center>
+          <Card
```
Comment (meharajM) [inline app/containers/Dashboard/index.js:48]: lets create small components and use them

File: app/containers/Login/index.js [hunk 5]
```diff
@@ -34,6 +34,21 @@ export function Login(props) {
   useInjectSaga({ key: 'login', saga });
   const auth = useAuth();
 
+  const signin = (user) => {
+    fetch('http://localhost:5000/api/auth/login', {
```
Comment (meharajM) [inline app/containers/Login/index.js:38]: lets keep all apis in separate file

File: mpp.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+"Logline \n\nTheme : Theme 1\nSub-Genre : Sub-Genre 1\nTitle : Title 1\n"
```
Comment (meharajM) [inline mpp.txt:1]: unwanted file

File: package.json [hunk 7]
```diff
@@ -67,13 +67,18 @@
     "@ant-design/icons": "^4.6.4",
     "@babel/polyfill": "7.4.3",
     "antd": "^4.16.13",
+    "blob": "^0.1.0",
```
Comment (meharajM) [inline package.json:70]: not needed

File: package.json [hunk 8]
```diff
@@ -67,13 +67,18 @@
...
     "chalk": "^2.4.2",
     "compression": "1.7.4",
     "connected-react-router": "6.4.0",
     "cross-env": "5.2.0",
+    "dotenv": "^10.0.0",
     "express": "4.16.4",
     "fontfaceobserver": "2.1.0",
+    "form-data": "^4.0.0",
```
Comment (meharajM) [inline package.json:78]: not needed

File: Title 1.txt (hunk 1 above)
Comment (shashankmotruri) [reply Title 1.txt:1]: removed

**Pull request merged**